serde_arrow/src/internal/schema/tracing_options.rs

```diff
@@ -49,6 +49,9 @@ pub struct TracingOptions {
     /// [`Strategy::MapAsStruct`][crate::schema::Strategy] for details.
     pub map_as_struct: bool,
 
+    /// If `true` serialize lists as LargeLists (the default).
```

Comment:
Is serialize correct here?

Reply:
I am somewhat inconsistent here. My latest preference is to use "tracing" for the detection of types. E.g., "trace lists as". 

If you're motivated: it would be cool, if you could change the language in the docs for `string` dictionary encoding?

Reply:
I'm leaving the doc change for other fields for another PR. There are possibly more fields which should be updated.